File: CGIworkspace2/GenerecDao/src/model/Personne.java
package model;

import java.util.Date;

public abstract class Personne {
	private long id = 0;
	private String nom;
	private String prenom;
	private Date dtNais;

	public Personne() {
		super();
	}

	public Personne(String nom, String prenom) {
		super();
		this.nom = nom;
		this.prenom = prenom;
	}

	public long getId() {
		return id;
	}

	public void setId(long id) {
		this.id = id;
	}

	public String getNom() {
		return nom;
	}

	public void setNom(String nom) {
		this.nom = nom;
	}

	public String getPrenom() {
		return prenom;
	}

	public void setPrenom(String prenom) {
		this.prenom = prenom;
	}

	public Date getDtNais() {
		return dtNais;
	}

	public void setDtNais(Date dtNais) {
		this.dtNais = dtNais;
	}

	@Override
	public String toString() {
		return "Personne [id=" + id + ", nom=" + nom + ", prenom=" + prenom + ", dtNais=" + dtNais + "]";
	}

}

File: CGIworkspaceFirstAngularApp/src/app/models/personne.ts

export class Personne {
    private _Email: string; 
    public get Email(): string {
        return this._Email;
    }
    public set Email(value: string) {
        this._Email = value;
    }


    private _Nom: string; 
    public get Nom(): string {
        return this._Nom;
    }
    public set Nom(value: string) {
        this._Nom = value;
    }


    private _Prenom: string; 
    public get Prenom(): string {
        return this._Prenom;
    }
    public set Prenom(value: string) {
        this._Prenom = value;
    }


    private _Password: string; 
    public get Password(): string {
        return this._Password;
    }
    public set Password(value: string) {
        this._Password = value;
    }


    private _Date: string;
    public get Date(): string {
        return this._Date;
    }
    public set Date(value: string) {
        this._Date = value;
    }
    private _Profil: string;
    public get Profil(): string {
        return this._Profil;
    }
    public set Profil(value: string) {
        this._Profil = value;
    }

    
  

}
